api/installation_service.go

```diff
@@ -52,3 +60,38 @@ func (is InstallationService) Export(outputFile string) error {
 
 	return nil
 }
+
+func (is InstallationService) Import(input ImportInstallationInput) error {
```

Comment:
Done...used SetupService to ensure non-availability before actually importing the installation.